lib/test/evaluation/tracker.py

```diff
@@ -81,9 +82,10 @@ def run_sequence(self, seq, debug=None):
 
         # Get init information
         init_info = seq.init_info()
-
+        print("tracker.run_sequence: init_info")
+        print(init_info)
         tracker = self.create_tracker(params)
-
+        print("after params")
```

Comment:
I suggest removing all debug prints unless they are important to understand results